axlearn/experiments/text/gpt/common.py

```diff
@@ -301,6 +305,10 @@ def model_config(
         lm_head=lm_head_cfg,
         dropout_rate=dropout_rate,
     )
+    if pad_token_id:
```

Comment:
Since `pad_token_id=0` is a valid token:
```suggestion
    if pad_token_id is not None:
```